apps/condo/domains/marketplace/schema/fields/price.js

```diff
@@ -0,0 +1,86 @@
+const Ajv = require('ajv')
+const addFormats = require('ajv-formats')
+
+const { render, getGQLErrorValidator } = require('@condo/domains/common/schema/json.utils')
+const { ERROR_INVALID_PRICE } = require('@condo/domains/marketplace/constants')
+const PRICE_GQL_TYPE_NAME = 'PriceSchemaField'
+const PRICE_GQL_INPUT_NAME = 'PriceSchemaFieldInput'
+
+const priceSchemaFields = {
+    type: 'String!',
+    group: 'String!',
+    name: 'String!',
+    price: 'String!',
+    isMin: 'Boolean!',
+    vat: 'String!',
+    salesTax: 'String!',
+}
+
+const priceGqlSchemaTypes = `
+    type ${PRICE_GQL_TYPE_NAME} {
+        ${render(priceSchemaFields)}
+    }
+    
+    input ${PRICE_GQL_INPUT_NAME} {
+        ${render(priceSchemaFields)}
+    }
+`
+
+const taxOptions = ['null', '0', '10', '20']
+const PRICE_FIELD_SCHEMA = {
+    type: 'object',
+    additionalProperties: false,
+    required: ['type', 'group', 'name', 'price', 'isMin', 'vat', 'salesTax'],
```

Comment:
VAT and sales tax are not required. We will use these values from InvoiceContext with the ability to override them here.
```suggestion
    required: ['type', 'group', 'name', 'price', 'isMin'],
```